packages/paste-website/src/pages/patterns/pricing/index.mdx

```diff
@@ -0,0 +1,482 @@
+import {UnorderedList, ListItem} from '@twilio-paste/list'
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {Box} from '@twilio-paste/box';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {ResponsiveImage} from '../../../components/ResponsiveImage';
+import PricingPatternCards from '../../../assets/images/patterns/pricing-pattern.png';
+
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Pricing');
+  return {
+    props: {
+      data: {
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+
+<NormalizedPatternHeader description={meta.description} name={meta.title} data={props.data} />
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+<ResponsiveImage src={PricingPatternCards} alt="Screenshot of a Pricing pattern cards example" />
+
+```jsx
+// import the components for pricing patterns as required
+
+import { Badge } from "@twilio-paste/core/badge";
+import { Heading } from '@twilio-paste/core/heading';
+import { Paragraph } from '@twilio-paste/core/paragraph';
+import { Text } from '@twilio-paste/core/text';
+import { Button } from '@twilio-paste/core/button';
+import { DisplayHeading } from '@twilio-paste/display-heading';
+import { Card } from '@twilio-paste/card';
+import { Box } from '@twilio-paste/box';
+import { VisualPickerRadio, VisualPickerRadioGroup } from "@twilio-paste/visual-picker";
+import { Separator } from "@twilio-paste/separator";
+import { SelectedIcon } from "@twilio-paste/icons/esm/SelectedIcon";
+```
+
+## About
+
+The Pricing pattern should be used when the user is presented with pricing or packaged options to select from. Use this pattern to clearly show the differences between different plans in order for the user to make an informed decision.
+
+Examples:
+<UnorderedList>
+  <ListItem>Selecting an account plan on upgrade.</ListItem>
+  <ListItem>Switching between a free and paid feature.</ListItem>
+  <ListItem>Adding a packaged plan to an account based on a use case.</ListItem>
+</UnorderedList>
+
+The information presented in the layout will be dependent on the packaged plans themselves.
+
+## General
+
+Pricing layouts should adhere to the following guidelines:
+<UnorderedList>
+  <ListItem>Limit the number of pricing Cards to 4, with a minimum of 2 as to not overwhelm users with options. For a larger quantity of plans, consider reducing the options or using a <Anchor href="/components/table">Table</Anchor>.</ListItem>
+  <ListItem>Display the current plan (if any).</ListItem>
+  <ListItem>Use a primary Button and Badge for the recommended option (if any), all other pricing cards use a secondary Button for selection.</ListItem>
```

Comment:
question(nit): should "cards" be capitalized or lowercase? seeing both here, let's stick to one or the other